### ISSUE TITLE
As a CentOS packager, I want my certificate-based auth workflow to continue working, so that I can continue contributing and being awesome.

### ISSUE THREAD

**Issue by sfinn85** [fedora-infra/noggin#34]

**Comment by abompard** [comment]
@relrod Same question as in #32, is this currently already handled by FreeIPA or FAS? Could you comment on this?

**Comment by puiterwijk** [comment]
This will need updating of the client tooling to talk to the IPA cert system instead of FAS.

**Comment by abompard** [comment]
OK, do you know who maintains the client tools in CentOS?

**Comment by abompard** [comment]
Maybe @arrfab has more details on that one?

**Comment by sfinn85** [comment]
I have reached out to Fabian @abompard

**Comment by arrfab** [comment]
@abompard : basically what our current tool (written by @bstinsonmhk) permits : letting users generate/request cert to Securitas/IPA from cli. (see https://wiki.centos.org/HowTos/CentosPackager)
The current CentOS tool (centos-packager) is/was maintained by @bstinsonmhk and is here : https://github.com/bstinsonmhk/centos-packager

Basically that was just the end-user cli tool ( https://github.com/bstinsonmhk/centos-packager/blob/master/SOURCES/centos-cert)  to call FAS (and the /dogencert) to get private key+cert combined into one .pem, and then use it for authentication (for example against kojihub)

So the way we were seeing it would be (and explained in the other ticket/user story): 
 * Securitas would expose API endpoint using IPA API in the backend (with the authentication token from the user authenticated through securitas) to let users request/get their CRT signed by dogtag/IPA
 * A cli tool for Securitas (I guess one is/should already be on its way for this) that would also automate that step (generate private key if not existing, generate .csr, use securitas endpoint , get back it signed as .crt)

Does that answer the question  ? basically permitting what FAS permits today

**Comment by sfinn85** [comment]
Thanks  Fabian ! @abompard  can you review when you get a chance and remove label if you are happy that we have all info required for the team to review?
Thank you :)

**Comment by sfinn85** [comment]
Needs to be done but not now, only after Fedora. Label to be added. lets look at labels...

**Comment by abompard** [comment]
Removing this issue from the backlog in favor of fedora-infra/fasjson#28, since it's probably a script that's going to fetch the certificates (and not the web UI).

**Comment by abompard** [comment]
Moved to fedora-infra/fasjson#28.
